terraform/addons/byo-firehose-logging-destination/target-account/variables.tf

```diff
@@ -27,16 +27,82 @@ variable "fleet_iam_role_arn" {
 }
 
 variable "results_prefix" {
-  default     = "results/"
+  default     = "results/year=!{timestamp:yyyy}/month=!{timestamp:MM}/day=!{timestamp:dd}/"
```

Comment:
This is for the sharding strategy we talked about previously on slack right?  We might want to document why this looks like this so that when it gets overridden by someone passing in a var they don't lose the benefit because they didn't know.